frontend/workflows/projectSelector/src/hello-world.tsx

```diff
@@ -84,17 +84,24 @@ const useDispatch = () => {
 
 const selectorReducer = (state: State, action: Action): State => {
   switch (action.type) {
-    case "ADD_PROJECTS":
-      // TODO: don't add if it already exists.
+    case "ADD_PROJECTS": {
```

Comment:
fyi wrapping the case in a `{}` b/c was getting the lint error `Unexpected lexical declaration in case block no-case-declarations` https://eslint.org/docs/rules/no-case-declarations

Reply:
i was planning on doing this anyways otherwise you can get some really weird bugs with variables being in scope but not actually existing between cases. can you go ahead and wrap all of the cases in `{ }` for consistency?